internal/nginx/maxmind.go

```diff
@@ -60,15 +73,53 @@ func GeoLite2DBExists() bool {
 
 // DownloadGeoLite2DB downloads the required databases by the
 // GeoIP2 NGINX module using a license key from MaxMind.
-func DownloadGeoLite2DB() error {
-	for _, dbName := range strings.Split(MaxmindEditionIDs, ",") {
-		err := downloadDatabase(dbName)
-		if err != nil {
-			return err
-		}
-		MaxmindEditionFiles = append(MaxmindEditionFiles, dbName+dbExtension)
+func DownloadGeoLite2DB(attempts int, period time.Duration) error {
+	if attempts < 1 {
+		attempts = 1
```

Comment:
you can set it to `MaxmindRetriesCount` variable, instead of hard-coding it.

Reply:
Thanks for that point. The only thing - we cannot use `MaxmindRetriesCount` because we are filling that argument with the value of according command line argument, and using it later as a first parameter in func `DownloadGeoLite2DB`. So I've added extra constant `const minimumRetriesCount = 1`, to avoid the hardcoding in the code.

Reply:
👍 yep. I missed it.